scifight/templatetags/scifight_url.py

```diff
@@ -0,0 +1,69 @@
+from django import template
+from scifight import models
+from django.core.urlresolvers import reverse, NoReverseMatch
+
+
+register = template.Library()
+
+
+@register.filter(is_safe=True)
+def scifight_url(model):
+    """
+    This filter simplifies the use of a template.
+    Input parameter is a model object.input parameter
+    is a model object.
+    {{ object | scifight_url }}
+    The filter will return the URL, from the corresponding
+    item in the urls.py. In case of an error NoReverseMatch
+    returns an empty string
+    """
+    def packed_reverse(*args, **kwargs):
+        try:
+            return reverse(*args, **kwargs)
+        except NoReverseMatch:
+            return ''
+
+    if isinstance(model, models.Team):
+        if model.slug:
+            kwargs = dict(team_slug=model.slug,
+                          tournament_slug= model.tournament.slug)
+            return packed_reverse('scifight:team_slug', kwargs=kwargs)
+        else:
+            kwargs = dict(team_id=model.id,
+                          tournament_slug=model.tournament.slug)
+            return packed_reverse('scifight:team_id', kwargs=kwargs)
+
```

Comment:
Did you notice that the below code is so similar? We may use the power of dynamic typing to get it short-n-sweet!
```python
if type(model) in {models.Participant, models.Leader, models.Jury, … }:
    name = type(model).__name__
    args = [ model.tournament.slug, model.id ]
    return wrapped_reverse("scifight:%s" % (type(model).__name__), args=args)
```
Or something like that ^__^.